src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -0,0 +1,56 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+import lombok.*;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
```

Comment:
Please do not use wildcard imports.
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad